docs/api/1.x/accounts.rst

```diff
@@ -0,0 +1,301 @@
+==============
+
+.. http:put:: /accounts/(user_id)
+
+    :synopsis: Creates a new account.
+
+    **Anonymous**
+
+    **Example Request**
+
+    .. sourcecode:: bash
+
+        $ echo '{"data": {"password": "azerty123"}}' | http PUT http://localhost:8888/v1/accounts/bob --verbose
+
+    .. sourcecode:: http
+
+        PUT /v1/accounts/bob HTTP/1.1
+        Accept: application/json, */*
+        Accept-Encoding: gzip, deflate
+        Connection: keep-alive
+        Content-Length: 36
+        Content-Type: application/json
+        Host: localhost:8888
+        User-Agent: HTTPie/0.9.8
+
+        {
+            "data": {
+                "password": "azerty123"
+            }
+        }
+
+    **Example Response**
+
+    .. sourcecode:: http
+
+        HTTP/1.1 201 Created
+        Access-Control-Expose-Headers: Backoff, Retry-After, Content-Length, Alert
+        Content-Length: 165
+        Content-Type: application/json
+        Date: Tue, 21 Mar 2017 14:30:14 GMT
+        Etag: "1490106614601"
+        Last-Modified: Tue, 21 Mar 2017 14:30:14 GMT
+        Server: waitress
+        X-Content-Type-Options: nosniff
+
+        {
+            "data": {
+                "id": "bob",
+                "last_modified": 1490106614601,
+                "password": "$2b$12$zlTlYet5v.v57ak2gEYyoeqKSGzLvwXF/.v3DGpT/q69LecHv68gm"
+            },
+            "permissions": {
+                "write": [
+                    "account:bob"
+                ]
+            }
+        }
+
+
+.. _accounts-udpate:
+
+Change password
+===============
+
+.. http:put:: /accounts/(user_id)
+
+    :synopsis: Changes the password for an existing account.
+
+    **Requires authentication**
+
+    **Example Request**
+
+    .. sourcecode:: bash
+
+        $ echo '{"data": {"password": "azerty123"}}' | http PUT http://localhost:8888/v1/accounts/bob --verbose --auth 'bob:azerty123'
+
+    .. sourcecode:: http
+
+        PUT /v1/accounts/bob HTTP/1.1
+        Accept: application/json
+        Accept-Encoding: gzip, deflate
+        Authorization: Basic Ym9iOmF6ZXJ0eTEyMw==
+        Connection: keep-alive
+        Content-Length: 36
+        Content-Type: application/json
+        Host: localhost:8888
+        User-Agent: HTTPie/0.9.2
+
+        {
+            "data": {
+                "password": "azerty123"
+            }
+        }
+
+    **Example Response**
+
+    .. sourcecode:: http
+
+        HTTP/1.1 200 OK
+        Access-Control-Expose-Headers: Backoff, Alert, Content-Length, Retry-After
+        Content-Length: 165
+        Content-Type: application/json
+        Date: Tue, 21 Mar 2017 17:11:58 GMT
+        Etag: "1490116321096"
+        Last-Modified: Tue, 21 Mar 2017 17:12:01 GMT
+        Server: waitress
+        X-Content-Type-Options: nosniff
+
+        {
+            "data": {
+                "id": "bob",
+                "last_modified": 1490116321096,
+                "password": "$2b$12$c12ui4O/z9gmVpGe1NMG2.Sb4zdw9p20oka2Seg3Xqq9rDpNR5HoW"
```

Comment:
Is it returned on purpose? If yes, what is it and how should I use it? I was naively thinking that this would remain hidden on the server.